image/preseed/preseed_linux.go

```diff
@@ -275,6 +279,29 @@ func prepareCore20Mountpoints(prepareImageDir, tmpPreseedChrootDir, snapdSnapBlo
 		{"--bind", writable, underPreseed("writable")},
 	}
 
+	if sysfsOverlay != "" {
+		// bind mount only permitted directories under sys/class and sys/devices
+		for _, dir := range []string{
+			"sys/class/backlight", "sys/class/bluetooth", "sys/class/gpio",
+			"sys/class/leds", "sys/class/ptp", "sys/class/pwm",
+			"sys/class/rtc", "sys/class/video4linux", "sys/devices/platform",
+			"sys/devices/pci0000:00"} {
```

Comment:
this should probably be a top-level var with a comment, also explaining that is meant to be in sync with what our interfaces use

Reply:
I moved it to the top as var with added comment